Main/cmd.py

```diff
@@ -1,80 +1,81 @@
+            os.system("mkdir %(out_dir)s %(annotate_path)s" % {"out_dir": args.out_dir,
+                                                               "annotate_path": path.join(args.out_dir, "annotate")})
+    elif os.path.exists(path.join(args.out_dir, "html")):
+        shutil.rmtree(path.join(args.out_dir, "html"))
 
-    db = DatabaseHandler(output_path, code_path, test_path)
+    db = DatabaseHandler(args.out_dir, args.module, args.tests)
 
+    # Convert list of modules directories to cov_modules string
     cov_modules = ""
-    for module in code_path:
+    for module in args.module:
         cov_modules += ("--cov=%(code_path)s " % {"code_path": module})
 
-    if not delete_out and os.path.exists(output_path):
-        output_path += str(db.get_last_run_id() + 1)
+    # Adding -- before each extra argument
+    extra_args = ""
+    for extra_arg in args.extra_args:
+        extra_args += ("--%(extra_arg)s " % {"extra_arg": extra_arg})
+
+    # Convert list of tests directories to tests string
+    tests = ""
+    for test_dir in args.tests:
+        tests += ("%(test_dir)s " % {"test_dir": test_dir})
 
-    exit_code = os.system(
-        "python -m pytest --cov-report annotate:%(cov_annotate)s --cov-report html:%(cov_html)s --cov-report xml:%(Covxml)s %(cov_modules)s"
-        " %(test_path)s --junitxml=%(Testsxml)s --html=%(pytest_report)s %(extra_args)s" % {
-            "cov_modules": cov_modules,
-            "test_path": test_path,
-            "Covxml": path.join(output_path, "coverage.xml"),
-            "Testsxml": path.join(output_path, "tests.xml"),
-            "cov_annotate": path.join(output_path, "annotate"),
-            "cov_html": path.join(output_path, "html"),
-            "pytest_report": path.join(output_path, "html", "pytest_report.html"),
-            "extra_args": extra_args})
+    if not args.delete_out and os.path.exists(args.out_dir):
+        args.out_dir += str(db.get_last_run_id() + 1)
+    pytest_cmd = "python -m pytest --cov-report annotate:%(cov_annotate)s --cov-report html:%(cov_html)s " \
+                 "--cov-report xml:%(Covxml)s %(cov_modules)s %(test_path)s --junitxml=%(Testsxml)s " \
+                 "--html=%(pytest_report)s %(extra_args)s" % \
+                 {
+                     "cov_modules": cov_modules,
+                     "test_path": tests,
+                     "Covxml": path.join(args.out_dir, "coverage.xml"),
+                     "Testsxml": path.join(args.out_dir, "tests.xml"),
+                     "cov_annotate": path.join(args.out_dir, "annotate"),
+                     "cov_html": path.join(args.out_dir, "html"),
+                     "pytest_report": path.join(args.out_dir, "html", "pytest_report.html"),
+                     "extra_args": extra_args
+                 }
 
+    exit_code = os.system(pytest_cmd)
     if exit_code != 0 and exit_code != 1:
         exit()
 
-    parser = Parser(db, output_path)
-    html = HTML(path.join(output_path, "html"), db)
```

Comment:
accidently removed this line also, line 71 shouldn't be deleted.

Reply:
You're right, I'll fix it and push again